eth/v1alpha1/beacon_chain.proto

```diff
@@ -759,6 +761,9 @@ message ValidatorAssignments {
         // Beacon chain slots in which the validator must perform its assigned
         // duty as a proposer.
         repeated uint64 proposer_slots = 4 [(gogoproto.casttype) = "github.com/prysmaticlabs/eth2-types.Slot"];
+        message ValidatorList {
```

Comment:
@mxmar Why do we need to add `ValidatorList` in existing `ValidatorAssignments` message?

Reply:
I've deleted it. My bad. Wrong place in the code...